app/script/calling/CallingRepository.js

```diff
@@ -65,7 +64,7 @@ z.calling.CallingRepository = class CallingRepository {
     this.conversationRepository = conversationRepository;
     this.mediaRepository = mediaRepository;
     this.userRepository = userRepository;
-    this.logger = new z.util.Logger('z.calling.CallingRepository', z.config.LOGGER.OPTIONS);
+    this.logger = new z.telemetry.calling.CallLogger('z.calling.CallingRepository', z.config.LOGGER.OPTIONS);
```

Comment:
Let's rename these to `callLogger` to make clear it has additional functionality whenever we use it.